ExampleMod/ExampleMod.csproj

```diff
@@ -5,8 +5,6 @@
     <AssemblyName>ExampleMod</AssemblyName>
     <LangVersion>latest</LangVersion>
   </PropertyGroup>
-  <ItemGroup>
-  </ItemGroup>
```

Comment:
A helpful cleanup, but please refrain from changes outside the scope of the PR in the future.